CHANGELOG.md

```diff
@@ -105,6 +105,9 @@
   Thanks [@PidgeyBE](https://github.com/PidgeyBE) for contributing this change.
   [#10595](https://github.com/Kong/kong/pull/10595)
   [#10204](https://github.com/Kong/kong/pull/10204)
+- Make `traditional_compat` mode more compatible to `traditional` by splitting routes with
+  multiple paths into multiple atc routes with separate priorities.
```

Comment:
Please elaborate on the impact on the end-user here.
The user should be able to read a single line and understand the change of behavior.
Also, please note this in the "breaking change/behavior" section since a user could already be relying on such behavior and this change could surprise them.